### PR TITLE
Closes # 168 udpated text around ANL02FL as alternative approach to the use of sou…

### PR DESCRIPTION
**Pull request by amitjaingsk**
…rce_pd parameter

Please add "Closes #<issue number>" to the title of the pull request. Then the
issue is closed automatically once it is merged to `main`.

Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiraldev/main/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/main/articles/unit_test_guidance.html#writing-unit-tests-in-admiral)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/main/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples 
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md`
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiralonco/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue so that it closes after successful merging. 
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done!  Much love to your accomplishment!

The following line requests to update the man pages by the "Man Pages" workflow.

/roxygenize

### CODE REVIEW

File: vignettes/adrs.Rmd
Comment (rossfarrugia) [inline vignettes/adrs.Rmd:238]:
```suggestion
So, for example the user could create `ANL02FL` flag to subset the post-baseline response data up to and including first reported progressive disease. This would be an alternative and transparent method to the use of `source_pd` argument approach to create ADRS parameters below.
```

Comment (rossfarrugia) [reply vignettes/adrs.Rmd:238]: i wouldn't mention NACT here again as we already give an example of that above.

**Review by rossfarrugia — CHANGES_REQUESTED** [review]
@amitjaingsk some minor comments/suggestions. looks good to me though! 

also reminder to tag the issue 168 on the right hand side development section of the PR, rename the PR title to include "closes #168 " at the start, and tick through all the boxes on the PR checklist above. i don't think this changes warrants anything in the NEWS.md/changelog personally, so no need to update that.

**Comment by rossfarrugia** [comment]
@bundfussr i guess we should also mention that `derive_var_relative_flag()` from https://github.com/pharmaverse/admiral/pull/1502 could be used here to derive `ANL02FL` in this case? 

FYI @amitjaingsk

**Comment by bundfussr** [comment]
@rossfarrugia , @amitjaingsk , I would also add an example call of `derive_var_relative_flag()` which flags all observations up to first PD.

**Comment by rossfarrugia** [comment]
Fine by me! My first thinking was to just explain in text but as we'll have a function for this now in {admiral} then no harm to include an example call.

File: vignettes/adrs.Rmd
Comment (amitjaingsk) [reply vignettes/adrs.Rmd]: @rossfarrugia updated the text and added the example for derive_var_relative_flag(). But there are some checks failing.

Comment (rossfarrugia) [reply vignettes/adrs.Rmd]: @amitjaingsk https://github.com/pharmaverse/admiral/pull/1502 needs to be merged first. will try and take a look at reviewing it for Stefan today. feel free to self-assign to that review too if you get chance

Comment (rossfarrugia) [reply vignettes/adrs.Rmd]: @amitjaingsk `derive_var_relative_flag()` is merged now in admiral latest devel branch

Comment (amitjaingsk) [reply vignettes/adrs.Rmd]: @rossfarrugia  thanks, I am still getting some eror due to some packages are out of sync. I have asked Matt to try to check from his system, so we will get you know.

File: vignettes/adrs.Rmd
Comment (rossfarrugia) [inline vignettes/adrs.Rmd:242]: @amitjaingsk FLASE is a typo and should be FALSE - try updating that

**Comment by rossfarrugia** [comment]
@amitjaingsk `derive_var_relative_flag()` is now ready in {admiral} latest `devel` branch, so can you make the final updates to this PR please so we can get it merged ahead of upcoming release? FYI @HinalPat

**Comment by mattmarino** [comment]
I looked into this for Amit, and there is an issue with installing admiralonco library. When I try to do it manually in rstudio I get the following error: `Error: object 'count_vals' is not exported by 'namespace:admiral'`. Do you know if there was a change to `count_vals` that would cause this?

**Comment by bundfussr** [comment]
@mattmarino , could you check which version of admiral you have installed? `count_vals()` was added to admiral 0.8.0.

File: vignettes/adrs.Rmd
Comment (bundfussr) [inline vignettes/adrs.Rmd:11]: What does this option do?

Comment (mattmarino) [reply vignettes/adrs.Rmd]: I am not seeing a difference with/without it, so I've removed.

**Comment by mattmarino** [comment]
> @mattmarino , could you check which version of admiral you have installed? `count_vals()` was added to admiral 0.8.0.

I managed to resolve this, thanks.

**Comment by mattmarino** [comment]
@bundfussr Can I ask you to take a look at the conflicts? The parts in devel look to be new vs what was published back in Sept so don't want to overwrite with something new if it should be there, and I'm not too sure.

**Comment by bundfussr** [comment]
@mattmarino , no problem, I have fixed the merge conflicts. The new parts from devel are correct.

**Comment by mattmarino** [comment]
@bundfussr Looks like my fix didn't solve it...I'm a bit stumped. If you have time, can I ask you to take a look at that item too?

**Comment by bundfussr** [comment]
@mattmarino , the R-CMD 4.0 is failing due to a general issue in the workflow. It is not related to the changes in this pull request. Thus we can ignore it.

**Review by rossfarrugia — APPROVED** [review]
@amitjaingsk @mattmarino i did a quick check and seems ok to me now so i added approval so that my previous review requests don't block you from merging this, but i would wait until you get approval from @bundfussr before merging this one

**Review by bundfussr — APPROVED** [review]

**Pull request merged**
